### PR TITLE
Updating to Julia v1.1, plus some other tricks

### PR DESCRIPTION
**Pull request by kraftpunk97-zz**
I was updating @nowozin's original repo and thought that this repo could use some love as well, since it's the one that is in the Julia's official registry. Here's a brief summary of the work I've done in this PR

- Updated 0.6-era elements to their v1.1 versions.

- wrote docstrings

- modified functions like `getScreen`, `getScreenRGB`, `getRAM` etc to work just with the ALE instance pointer, instead of requiring a Vector argument as well with the right length to make the ccall work.

- A few more things that I can't remember off the top of my head.

### CODE REVIEW

**Comment by jbrea** [comment]
Awesome, thanks a lot!

For docstrings I usually try to follow the [recommendations](https://docs.julialang.org/en/v1/manual/documentation/), in particular point 1, i.e. "always show the signature of a function at the top of the documentation". Is it intentional that you omitted this?

Also, there are currently still some conflicts in build.jl and aleinterface.jl

**Comment by jbrea** [comment]
Ps. I added a comment in the README that this is the maintained fork that is in the registry, but maybe it would be good to indicate this also in @nowozin's repo.

**Comment by findmyway** [comment]
Also, it would be great to add some tests for this repo. The last time I did some benchmark tests, I found many environments in this repo were broken (a weird segment fault) except "pong". 

See https://github.com/JuliaReinforcementLearning/ReinforcementLearningEnvironments.jl/blob/master/benchmarks/speed_of_random_action.jl#L45

**Comment by nowozin** [comment]
FYI, I just added an obsoletion notice to the top of my old repository, see [here](https://github.com/nowozin/ArcadeLearningEnvironment.jl)

**Comment by jbrea** [comment]
Thanks a lot @nowozin!

**Comment by jbrea** [comment]
@Jun I added some tests for each ROM in 300ff5897567e9632783a14b821ea342c4f56b83. It looks like all ROMs are fine, except "defender.bin".

**Comment by kraftpunk97-zz** [comment]
@jbrea I will fix the docstrings. Thanks for the reference link.

@nowozin If that repository is being archived, is it okay if I close my PR and push those changes through this PR instead?

**Comment by kraftpunk97-zz** [comment]
@jbrea @findmyway The Travis jobs are failing because I modified the prototype of `getScreen`, `getScreenRGB` and `getScreenGrayscale`. I modified runtests.jl to reflect those changes, and the tests passed on my system.

I've modified the docstrings as asked. The tests for this package seem reasonable to me. Can't think of anything else.

File: deps/build.jl [hunk 1]
```diff
@@ -50,15 +50,17 @@ if libale_detected == false
             CreateDirectory(_libdir)
             @build_steps begin
                 ChangeDirectory(_srcdir)
-                `rm -rf Arcade-Learning-Environment-0.6.0`
+                `rm -rf Arcade-Learning-Environment-0.5.2`
+                `rm -rf v0.5.2.zip`
+				`rm -rf Arcade-Learning-Environment-0.6.0`
                 `rm -rf v0.6.0.zip`
-                `curl -LO https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.6.0.zip`
+                `wget https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.6.0.zip`
```
Comment (jbrea) [inline deps/build.jl]: I would use curl here, see e.g. [here](https://github.com/JuliaLang/julia/issues/374).

**Comment by jbrea** [comment]
Thanks a lot, this looks really great now.
I would just use curl instead of wget, see above.

File: deps/build.jl (hunk 1 above)
Comment (kraftpunk97-zz) [reply deps/build.jl]: Oh I see. I thought wget was shipped by default and curl had to be downloaded. Fixing it right away...

Comment (kraftpunk97-zz) [reply deps/build.jl]: @jbrea ✅ done

**Comment by jbrea** [comment]
Thanks @kraftpunk97 :+1:

**Pull request merged**